nRF9160Master/src/WiFi/WiFiHandler.c

```diff
@@ -258,6 +260,29 @@ static void CheckAPConnected(const char *pcResp, bool *pbStatus)
     } 
 }
 
+/**
+ * @brief      : GetAPCredentials
+ * @param [in] : None
+ * @param [out]: None
+ * @return     : WIFI_SSID_PWD
+*/
+char *GetAPCredentials(void)
+{
+    return WIFI_SSID_PWD;
```

Comment:
Consider naming of this string variable. Use something like as shown in example. We will be using this type of naming for MACROS
for Eg: cWifiCredentials